ui/app/src/state/epics/pathNavigatorTree.ts

```diff
@@ -338,14 +338,18 @@ export default [
             [parentPathOfTargetPath, parentPathOfSourcePath].forEach((path) => {
               if (
                 // If in totalByPath is an item that has been loaded and must update...
-                path in tree.totalByPath
+                // path in totalByPath may be a page, and its path has index.xml, so it needs to be validated too.
+                path in tree.totalByPath ||
+                withIndex(path) in tree.totalByPath
```

Comment:
Does it make sense to assure to comparison is made both ways?
```suggestion
                withoutIndex(path) in tree.totalByPath ||
                withIndex(path) in tree.totalByPath
```

Reply:
`path` is one of the parents (parentPathofTargetPath or parentPathOfSourcePath), and they're retrieved using `getParentPath`. This is the same scenario as the one above, where `getParentPath` already removes the index.xml from the path